frontends/p4/uniqueNames.cpp

```diff
@@ -80,8 +80,8 @@ addNameAnnotation(cstring name, const IR::Annotations* annos) {
 }
 
 UniqueNames::UniqueNames(ReferenceMap* refMap) : renameMap(new RenameMap) {
-    setStopOnError(true);
     setName("UniqueNames");
+    visitDagOnce = false;
```

Comment:
Won't this create multiple unique names for a single shared object?  Thus turning it into multiple objects?